Getting started with Amundsen

don-simpson opened the pull request:
CDK Project for blog: Getting started with Amundsen leveraging Amazon Neptune and the AWS CDK

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

triggan reviewed (commented):
Let's get a bit more detail into the README.md to cover the deployment for those that may not see the blog post.  Also, let's modify the final stack to output the URL for Amundsen's console.   Once both of those are done, this should be ready to ship.  I deployed into my account this morning and it worked without any major issues.  Only issues were on my end (didn't have CDK installed and did not have an initial CDK environment created).  I noted both of those stumbles in the blog post feedback.  It would be good to address those items in case other users come across them.

don-simpson commented:
> Let's get a bit more detail into the README.md to cover the deployment for those that may not see the blog post. Also, let's modify the final stack to output the URL for Amundsen's console. Once both of those are done, this should be ready to ship. I deployed into my account this morning and it worked without any major issues. Only issues were on my end (didn't have CDK installed and did not have an initial CDK environment created). I noted both of those stumbles in the blog post feedback. It would be good to address those items in case other users come across them.

Updated the README and modified the stack to output the Amundsen Frontend Hostname.

triggan reviewed (approved):
Looks good!  Approved for commit.
